alot/db/message.py

```diff
@@ -54,6 +54,11 @@ def __init__(self, dbman, msg, thread=None):
         self._attachments = None  # will be read upon first use
         self._tags = set(msg.get_tags())
 
+        self._session_keys = []
+        for name, value in msg.get_properties("session-key", exact=True):
+            if "session-key" == name:
```

Comment:
Can you swap the constant and the variable here?

Reply:
done